e2e/runner/start-sample-app-containers/start-sample-app-containers.ts

```diff
@@ -0,0 +1,67 @@
+import { FAILURE_EXIT_CODE } from "../constants/exit-code";
+import { printBold } from "../cypress-runner-utils";
+import { SAMPLE_APP_SETUP_CONFIGS } from "../sample-apps-shared/constants/sample-app-setup-configs";
+import { fetchApp } from "../sample-apps-shared/helpers/fetch-app";
+import {
+  copyExampleEnvFile,
+  copyLocalEmbeddingSdkPackage,
+  copyLocalMetabaseJar,
+} from "../sample-apps-shared/helpers/prepare-app";
+import { startContainers } from "../sample-apps-shared/helpers/start-containers";
+import type {
+  EmbeddingSdkVersion,
+  SampleAppTestSuiteName,
+} from "../sample-apps-shared/types";
+
+const userOptions = {
+  EMBEDDING_SDK_VERSION: "local",
+  ...process.env,
+} as const;
+
+printBold(`Running Cypress Sample App Tests with options:
+  - EMBEDDING_SDK_VERSION      : ${userOptions.EMBEDDING_SDK_VERSION}
+`);
+
+export async function startSampleAppContainers(
+  testSuite: SampleAppTestSuiteName,
+) {
+  const embeddingSdkVersion =
+    userOptions.EMBEDDING_SDK_VERSION as EmbeddingSdkVersion;
+  const setupConfig = SAMPLE_APP_SETUP_CONFIGS[testSuite];
+  const {
+    appName,
+    branch,
```

Comment:
this `branch` is from 
https://github.com/metabase/metabase/blob/a71ceea29649d52a1e6324f1ca04473edc327b7b/e2e/runner/sample-apps-shared/constants/sample-app-setup-configs.js#L1

do we need to update that once we merge those branches on each of the sample app?

Reply:
Yes. I will merge a sample app pr first, and then adjust this value before merging the PR